script/migrate-test-e2e.js

```diff
@@ -33,17 +33,47 @@ const guideTitle = "Contributing to a create-typescript-app Repository";
 const owner = "JoshuaKGoldberg";
 const title = "Create TypeScript App";
 
+const originalDevelopment = (
+	await fs.readFile(".github/DEVELOPMENT.md")
+).toString();
+
 await $({
 	stdio: "inherit",
 })`c8 -o ./coverage -r html -r lcov --src src node ${bin} --base everything --author ${authorName} --mode migrate --bin ${bin} --description ${description} --email-github ${emailGithub} --email-npm ${emailNpm} --guide ${guide} --guide-title ${guideTitle} --owner ${owner} --title ${title} --repository ${repository} --skip-all-contributors-api --skip-github-api --skip-install`;
 
-test.each([...filesExpectedToBeChanged])("verify %s", async (file) => {
-	const { stdout } = await execaCommand(`git diff HEAD -- ${file}`);
-	expect(stdout.split("\n").slice(2).join("\n")).toMatchSnapshot();
+// All Contributors seems to not be using Prettier to format files...
```

Comment:
💡 I wonder if it's that this repo's prettier config has plugins that need to be installed!